runtime/src/iree/hal/cts/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ set(IREE_ALL_CTS_TESTS
   "command_buffer_fill_buffer"
   "command_buffer_update_buffer"
   "driver"
+  "device_group_copy"
```

Comment:
Device groups aren't a HAL concept and not something we want to expose like this - these tests only seem to be exercising multiple queues on the same device and that's something all devices should support regardless of whether they are backed my a single physical device or multiple ones. I see "gpu0" "gpu1" "device0" "device1" etc in there that should just be "queue0" and "queue1".

These should really just be part of the normal test suite. All HAL targets should be able to support correct execution of code against multiple queues even if multiple aren't available though we may have bugs and some may want to be excluded (synchronous CPU _should_ work but may not in some cases today - that's a bug, though). We could either exclude them by naming them uniquely and dealing with the build system or by using a device query to no-op the test if multiple queues aren't available. A device query for `hal.device`/`concurrency` should return 1 if there's only a single queue and >1 otherwise. If a particular test only works with a certain queue count (wait-before-signal on work from another queue, etc) then we could `GTEST_SKIP` it.